com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -168,6 +202,29 @@ internal void AddSpawn(SnapshotSpawnCommand command)
             }
         }
 
+        internal void AddDespawn(SnapshotDespawnCommand command)
+        {
+            if (NumDespawns >= m_MaxDespawns)
+            {
+                Array.Resize(ref Despawns, 2 * m_MaxDespawns);
+                m_MaxDespawns = m_MaxDespawns * 2;
+                Debug.Log($"[JEFF] despawn size is now {m_MaxDespawns}");
+            }
+
+            if (NumDespawns < m_MaxDespawns)
+            {
+                if (command.TargetClientIds == default)
+                {
+                    command.TargetClientIds = GetClientList();
```

Comment:
This allocates each spawn/despawn. Don't think we can fix this here quickly. Just pointing out that we are again adding a significant amount of allocations.

Reply:
Yes. This is unfortunate, but hopefully bearable for "small scale coop". The plan is to replace this with a bit mask, but I'm not seeing this making it in time for the next release.